mnelab/__main__.py

```diff
@@ -27,7 +28,11 @@ def _run():
                 if info:
                     info["CFBundleName"] = app_name
 
-    matplotlib.use("Qt5Agg")
+    if qtpy.PYQT5 or qtpy.PYSIDE2:
+        matplotlib.use("Qt5Agg")
+    elif qtpy.PYQT4 or qtpy.PYSIDE:
+        matplotlib.use("Qt4Agg")
+
```

Comment:
I think you can remove these changes (and `import qtpy`), because MNELAB doesn't support the old PyQt4/PySide packages.